src/main/features/response/views/macro/check-and-send/admission.njk

```diff
@@ -0,0 +1,67 @@
+{% from "table.njk" import tableStart, tableEnd, row %}
+
+{% macro whenWillYouPayRow(value, url) %}
+  {{
+    row(
+      label = 'When will you pay',
+      value = value,
+      changeLink = url,
+      bold = true
+    )
+  }}
+{% endmacro %}
+
+{% macro notImmediatePayment(type, paymentDate, paymentPlan, explanation, Paths = {}, externalId) %}
+  {{ tableStart('whenWillYouPay', 'When you’ll pay') }}
+  {% set whenWillYouPayLabel = t(type.option.displayValue) | capitalize %}
+
+  {% if type.isOfType(DefendantPaymentType.BY_SET_DATE) %}
+    {%
+      set whenWillYouPayLabel = t('{{ whenWillYouPayLabel }}: {{ date }}', {
```

Comment:
That different compared what we had before. I do not think we ever had `{{ whenWillYouPayLabel }}: {{ date }}` in label column.

Reply:
it's what ronan asked for, he's updating the prototype to match